account_manager/src/validator/exit.rs

```diff
@@ -0,0 +1,440 @@
+use crate::wallet::create::STDIN_INPUTS_FLAG;
+use crate::{SECRETS_DIR_FLAG, VALIDATOR_DIR_FLAG};
+use bls::{Keypair, PublicKey};
+use clap::{App, Arg, ArgMatches};
+use directory::{parse_path_or_default_with_flag, DEFAULT_SECRET_DIR};
+use environment::Environment;
+use eth2::{
+    types::{GenesisData, StateId, ValidatorId, ValidatorStatus},
+    BeaconNodeHttpClient, Url,
+};
+use eth2_testnet_config::Eth2TestnetConfig;
+use safe_arith::{ArithError, SafeArith};
+use std::path::PathBuf;
+use std::time::{SystemTime, UNIX_EPOCH};
+use types::{ChainSpec, Epoch, EthSpec, Fork, Hash256, Slot, VoluntaryExit};
+use validator_dir::{Manager as ValidatorManager, ValidatorDir};
+
+pub const CMD: &str = "exit";
+pub const VALIDATOR_FLAG: &str = "validator";
+pub const BEACON_SERVER_FLAG: &str = "beacon-node";
+pub const PASSWORD_PROMPT: &str = "Enter the keystore password";
+
+pub const DEFAULT_BEACON_NODE: &str = "http://localhost:5052/";
+pub const CONFIRMATION_PHRASE: &str = "Exit my validator";
+pub const WEBSITE_URL: &str = "https://lighthouse-book.sigmaprime.io/voluntary-exit.html";
+pub const PROMPT: &str =
+    "WARNING: WITHDRAWING STAKED ETH WILL NOT BE POSSIBLE UNTIL ETH1/ETH2 MERGE";
+
+pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
+    App::new("exit")
+        .about("Submits a VoluntaryExit to the beacon chain for a given validator(s).")
+        .arg(
+            Arg::with_name(VALIDATOR_FLAG)
```

Comment:
I would be tempted to just accept a path to an EIP-2335 keystore and always prompt for the password, instead of dealing with the validator dirs. It'll work in a broader set of cases and will be less code for us to maintain.

Now we've done (are doing) away with `lighthouse account validator deposit`, we don't need to follow its lead regarding this flag anymore.

Reply:
Yes I would like this very much. Will make the changes.

Reply:
What do you think about an option for providing a keystore password file. If no file is provided, then we prompt for a password. Could be easier for people trying to exit multiple validators with different passwords.